documentation/_getting-started.pug

```diff
@@ -257,6 +257,17 @@ a(href='https://github.com/eliep/vue-avatar')
                   initials="AS"
                   :size="100"&gt;
                 &lt;/avatar&gt;
+        tr
+          td
+            avatar(username='Luke Skywalker', src='./static/luke-skywalker.png', :size='100')
+          td
+            pre
+              code.language-html.
+                &lt;!-- fallback to initials if image doesn't load --&gt;
+                &lt;avatar
+                  src="./static/luke-skywalker.png"
+                  :size="100"&gt;
```

Comment:
Could you also add the `username` prop to the example here? Otherwise it's unclear where the `LS` initials come from in the example